snap/local/opt/kafka/bin/bin-wrapper.bash

```diff
@@ -3,9 +3,7 @@
 set -e
 
 unset KAFKA_JMX_OPTS
+unset KAFKA_LOG4J_OPTS
 
-if [ "x$KAFKA_LOG4J_OPTS" = "x" ]; then
-    export KAFKA_LOG4J_OPTS="-Dlog4j.configuration=file:${SNAP_DATA}/etc/kafka/tools-log4j.properties"
-fi
```

Comment:
**todo** why is this needed? I don't think we should do this. We need this to make sure that CLI commands are logged out at console, such that when charm does something we get relevant log in the charm logs

Reply:
I don't agree. Right now, we get hundreds of lines of Java logging, for a single output line. It's been really painful for our users who have manually `unset` the env-var for their deployments.

We don't need the full Java logs for bin-commands, just sysout.

Reply:
**issue** If we get rid of these lines, which log4j configuration would we use for any commands we run with the snap? The reason for these lines to be there is that when running the Kafka bin commands with the snap one would use the `tools-log4j.properties` as for upstream. However, differently from upstream, this file is in another directory now (since the default one would be in the read-only squashfs)

It is also what we do in the other scripts

Reply:
I agree with Marc here. For bin commands it's quite ugly to show a diarrhea of Java code which adds +100 lines at times when the output should be a simple line. That's why `--verbose` flags exist on CLIs.

Maybe we can implement a `--verbose` flag, and check for that on this script?